src/main/java/com/nerodesk/Launch.java

```diff
@@ -73,6 +73,20 @@ public static void main(final String... args) throws Exception {
     }
 
     /**
+     * Return passed port or run on default one 8080.
+     * @param args Command line args
+     * @return Application port.
+     */
+    private static int port(final String... args) {
```

Comment:
do we really need to extract this code in a separate method?
I could say YES if:
- we are going to reuse it
- main method is too big (say, over 30 lines)

In this case, I would rather inline it